templates/helper_spec_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module HelperSpecHelpers
+  def stub_current_user(as: nil)
```

Comment:
Should this be part of Clearance's RSpec support? I think we should try to keep Suspenders as just configuration whenever possible. Otherwise, we're essentially copying and pasting code into every project.